docker-compose-proxy.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.3'
```

Comment:
Well, what about [this](https://github.com/docker/compose/blob/77e7989eaaad1addde5c84d15ace61aeaa14ab96/ROADMAP.md#more-than-just-development-environments) then?